contrib/packaging/rpm/skydive.te

```diff
@@ -0,0 +1,99 @@
+policy_module(skydive, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type skydive_t;
+type skydive_exec_t;
+init_daemon_domain(skydive_t, skydive_exec_t)
+
+permissive skydive_t;
```

Comment:
This shouldn't be part of the policy, but rather set temporarily using "semanage permissive -a skydive_t"